pages/common/grunt.md

```diff
@@ -0,0 +1,35 @@
+# grunt
+
+> A JavaScript task runner for automating processes.
+
+- Run the default task process:
+
+`grunt`
+
+- Run one or more specific space-separated task(s):
+
+`grunt {{task_name}}`
+
+- Specify an alternative configuration file:
+
+`grunt --gruntfile {{path/to/file}}`
+
+- Specify an alternative base path for relative files:
+
+`grunt --base {{path/to/directory}}`
+
+- Specify an additional directory to scan for tasks in:
+
+`grunt --tasks {{path/to/directory}}`
+
+- Specify a local npm package name to scan for tasks in:
```

Comment:
This seems like a corner case to me. Not sure if we want to include it. Thoughts ?

Reply:
I don't use grunt, so I'm not a particularly good person to ask, but I'm a bit confused as to what this actually _does_ specifically?

Reply:
This is basically for if say, the `grunt-copy-stuff` package has written some tests that you want to include in your grunt script, you can specify `--npm grunt-copy-stuff` and it will include their tests in yours.

To be honest, I've hardly used this option.

Reply:
pinging grunt owners @vladikoff / @shama - Is the `--npm` option something a newcomer is needed to know ?

Reply:
> Is the --npm option something a newcomer is needed to know ?

nope!

Reply:
:+1: Cheers @vladikoff. I'll remove it to be easier for new users.